UnityProject/Assets/Scripts/IMGUIDemo.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using ImGuiNET;
+using ImGuiNET.Unity;
+using UnityEngine;
+
+public class IMGUIDemo : MonoBehaviour
+{
+	[SerializeField] private bool showDemo = false;
+	private void Update()
+	{
+		if (Input.GetKey(KeyCode.LeftShift) && Input.GetKeyDown(KeyCode.Insert))
+		{
+			showDemo = !showDemo;
+			if (showDemo)
+			{
+				ImGuiUn.Layout += OnLayout;
+			}
+			else
+			{
+				ImGuiUn.Layout -= OnLayout;
```

Comment:
Do you need to allow this on the code scan list?

Reply:
Oh yeah thanks for reminding me.
this needs to be allowed on the code scan list